app/resources/device.py

```diff
@@ -237,3 +239,40 @@ def owner(data: dict, microservice: str) -> dict:
         return device_not_found
     else:
         return {"owner": device.owner}
+
+
+@m.microservice_endpoint(path=["delete_user"])
+def delete_user(data: dict) -> dict:
```

Comment:
Can you leave the parameter `microservice: str` out without any problems?

Reply:
No, the function has to accept two parameters: `data` and `microservice`

Reply:
So please add that @TheCataliasTNT2k